kafka-connect/kafka-connect-events/src/main/java/org/apache/iceberg/connect/events/CommitReadyPayload.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.connect.events;
+
+import java.util.List;
+import java.util.UUID;
+import org.apache.avro.Schema;
+import org.apache.avro.SchemaBuilder;
+import org.apache.iceberg.avro.AvroSchemaUtil;
+
+/**
+ * A control event payload for events sent by a worker that indicates it has finished sending all
+ * data for a commit request.
+ */
+public class CommitReadyPayload implements Payload {
+
+  private UUID commitId;
+  private List<TopicPartitionOffset> assignments;
+  private final Schema avroSchema;
+
+  private static final Schema AVRO_SCHEMA =
+      SchemaBuilder.builder()
+          .record(CommitReadyPayload.class.getName())
+          .fields()
+          .name("commitId")
+          .prop(AvroSchemaUtil.FIELD_ID_PROP, 1100)
```

Comment:
I notice that we're using a wide range of field identifier values for the same fields.  Is this because we're trying to deconflict with other Payloads/Elements (e.g. 1100, 1200, 1300)?  I can see the need if the avro schema embeds another element (like `TableName`).  Just wondering if there's anything we would really need to be concerned about by having a more explicit/consistent mapping (e.g. `FIELD_ID_PROP = 1000` everywhere).

Reply:
I originally had it that way, with the same prop ID everywhere, as it isn't used for anything. I changed this to have unique field IDs to address one comment that having real field IDs would be helpful. I'm OK either way.

Reply:
I'm fine with it, just wasn't sure if there was a reason behind the numbering.